contributing/content-style.md

```diff
@@ -102,12 +102,12 @@ Some guides to helping you make sure your writing is inclusive and accessible to
 
 To make your content as accessible as possible, the purpose of a link should be clear from its text alone.
 Avoid links with text like `click here`
-and instead try to include meaning inside the link itself.
+and instead, try to include meaning inside the link itself.
```

Comment:
:clipboard: The comma there does not fit with our style. We do not want to emphasize it as an aside.

```suggestion
and instead try to include meaning inside the link itself.
```